pkg/doc/verifiable/test-suite/verifiable_suite_test.go

```diff
@@ -79,9 +77,25 @@ func encodeVCToJWS(vcBytes []byte, privateKey interface{}) {
 
 	jwtClaims, err := credential.JWTClaims(true)
 	if err != nil {
-		abort("verifiable credential encoding to JWT failed: %v", err)
+		abort("verifiable credential encoding to JWS failed: %v", err)
+	}
+
+	jws, err := jwtClaims.MarshalJWS(verifiable.RS256, privateKey, "any")
+	if err != nil {
+		abort("failed to serialize JWS: %v", err)
+	}
+
+	fmt.Println(jws)
```

Comment:
remove Println

Reply:
This is required for this app. It's actually an app and not a test. See explanation at the bottom of the file.